nodejs/v3/nestjs-prisma/app/appsignal.cjs

```diff
@@ -0,0 +1,7 @@
+const { Appsignal } = require("@appsignal/nodejs");
+
+new Appsignal({
+  active: true,
+  name: "nestjs-prisma-screenshot",
+  pushApiKey: "not-a-real-api-key",
```

Comment:
Might be best to not specify the `pushApiKey` config option at all, as it should come for the environment. That way, if for some reason it was missing from the environment, it should error.

Reply:
I'll clean that up to all setups in another PR